tests/test_web_poet_rules.py

```diff
@@ -457,25 +454,6 @@ def name(self) -> str:
         return "wrong url"
 
 
-@inlineCallbacks
-def test_basic_item_with_page_object_but_different_url() -> None:
```

Comment:
The current code doesn't raise a warning. We discussed the old code with @kmike and decided that raising a warning is incorrect in this use case.